docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,533 @@
+===========================================
 
-The crate-node CLI enables you to perform certain unsafe operations
-on a node that are only possible while it is shut down. This command allows you
-to adjust the role of a node and may be able to recover some data after a
-disaster.
+This section shows you how to execute certain troubleshooting operations on
+CrateDB nodes and clusters using the ``crate-node`` command, including:
 
-If a CrateDB cluster unrecoverably looses the majority of the master eligible
-nodes, the crate-node CLI provides a way to utilize the state of the remaining
-master nodes to form a new cluster.
+* changing the states of remaining master nodes to form a new cluster,
+* bootstrapping a cluster to override its voting configuration, and
+* detaching a node from its cluster, if a cluster is not recoverable.
 
-Nodes may be temporarily stopped for maintenance reasons or systems
-failures. Once the issues on the nodes are resolved, a restart is necessary
-to let them join the cluster again. Unfortunately this is not always
-possible. A node might be unrepairable and does not start-up,
-so if the cluster is still available the node can be simply replaced by
-a fresh node on a new host which will replace the broken node.
+.. rubric:: Table of contents
 
-crate-node CLI
-==============
+.. toctree::
+   :maxdepth: 1
 
-.. code-block:: console
+.. contents::
+   :local:
 
-   bin/crate-node repurpose|unsafe-bootstrap|detach-cluster
-   [--ordinal <Integer>] [-C <KeyValuePair>]
-   [-h, --help] ([-s, --silent] | [-v, --verbose])
+Changing the role of a node
+===========================
 
+Each CrateDB data node maintains the following data on disk:
 
-The following commands provide a various operations to modify and recover nodes:
+* the shard data for every shard allocated to that node,
+* the index metadata corresponding with every shard allocated to that node, and
+* the cluster-wide metadata, such as settings and index templates.
 
-* ``crate-node repurpose`` can be used to delete unwanted data from a node if it
-  used to be a data node or a master-eligible node but has been repurposed not to
-  have one or other of these roles.
+Similarly, each CrateDB  master-eligible node maintains the following data on
+disk:
 
-* ``crate-node unsafe-bootstrap`` can be used to perform unsafe cluster bootstrapping.
-  It forces one of the nodes to form a brand-new cluster on its own, using its local
-  copy of the cluster metadata.
+* the index metadata for every index in the cluster, and
+* the cluster-wide metadata, such as settings and index templates.
 
-* ``crate-node detach-cluster`` enables you to move nodes from one cluster to another.
-  This can be used to move nodes into a new cluster created with the
-  ``crate-node unsafe-bootstap`` command. If unsafe cluster bootstrapping was not
-  possible, it also enables you to move nodes into a brand-new cluster.
+Now, the `repurpose command`_ lets you delete any of a node's shard data,
```

Comment:
change tense. "When you run the repurpose command, "

in fact, start this section with an intro to the repurpose command. and then add this stuff about deleting node data as a caveat / follow-on. at the moment. we just open with some facts about node data, without indicating why the reader should care

Reply:
try to avoid possessive's with nouns like "node"

Reply:
Deleting data from a node is the command's main purpose. I took over and rephrased the elasticsearch intro now. The intro does not beat around the bush but jumps right into what the command does (delete data from nodes). What do you think?

I've nixed all possessives.